meta/main.yml

```diff
@@ -3,7 +3,7 @@ galaxy_info:
   company: Idealista S.A.U.
   author: Idealista S.A.U.
   description: Apache ZooKeeper role
-  min_ansible_version: 2.8.1
+  min_ansible_version: 2.8.6
```

Comment:
whats the reason of this change? 2.8.1 is not compatible?

Reply:
Because we aren't testing that version with Travis, and we can't ensure backwards compatibility. I prefer to specify only versions that we can check that works using Molecule.